sdk/include/hipdnn_sdk/test_utilities/FlatbufferDatatypeMapping.hpp

```diff
@@ -0,0 +1,100 @@
+// Copyright © Advanced Micro Devices, Inc., or its affiliates.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <hipdnn_sdk/data_objects/data_types_generated.h>
+#include <hipdnn_sdk/utilities/UtilsBfp16.hpp>
+#include <hipdnn_sdk/utilities/UtilsFp16.hpp>
+
+namespace
+{
+template <typename T>
+struct TypeWrapper
+{
+    using type = T;
```

Comment:
Do you have a naming convention? If classes have UpperCase names, why would an alias to a type be lower case? I just want to make sure I understand this project's style guide.